commercetools-models/src/main/java/io/sphere/sdk/subscriptions/MessageSubscription.java

```diff
@@ -0,0 +1,21 @@
+package io.sphere.sdk.subscriptions;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.sphere.sdk.annotations.ResourceValue;
+
+import javax.annotation.Nullable;
+import java.util.Arrays;
+import java.util.List;
+
+@JsonDeserialize(as = MessageSubscriptionImpl.class)
+@ResourceValue
+public interface MessageSubscription {
+    String getResourceTypeId();
+
+    @Nullable
+    List<String> getTypes();
+
+    static MessageSubscription of(final String resourceTypeId, final String... types) {
```

Comment:
I would avoid using varargs or I would have it as syntactic sugar for `of(final String resourceTypeId, final List<String> types)`. Having it as the only method forces developers to use ugly arrays if they are providing types from a list.